### ISSUE TITLE
Support for lxqt-wayland

### ISSUE THREAD

**Issue by twoda5h** [WayfireWM/wcm#91]
Hi, when running lxqt with wayfire as the compositor, the configuration is stored in -> `$XDG_CONFIG_HOME/lxqt/wayland/`
Is there any way to make wcm read this directory for the config so i don't have to edit it manually?

**Comment by soreau** [comment]
Yes, you can set `WAYFIRE_CONFIG_FILE` to the path of `wayfire.ini` when running wcm.
